lib/std/crypto/blake2.zig

```diff
@@ -69,7 +69,7 @@ pub fn Blake2s(comptime out_len: usize) type {
         key: []const u8,
 
         pub fn init() Self {
-            return init_keyed("");
+            return comptime init_keyed("");
```

Comment:
This is pretty similar to the `inline` situation. Better to not do this unless you have some good reason to do it. The optimizer will figure this out, and you're actually restricting what the compiler is allowed to do here.

Reply:
Remove, thank you!